data/game-documents/fari-games/charge-rpg.md

```diff
@@ -1512,6 +1512,10 @@ Depending on what is being repaired, the player may have to make different `acti
 
 Once the project is completed, the character is re-allowed to use the attachment it in the fiction, and may use it again to block a new consequence.
 
+## Attachment Condition 
+
+Maybe you want an attachment that degrades over time, with use or by soaking up consequences. You can do so by introducing the `Consequence` mechanic from the character sheet on your attachment. Then, the more damage your attachment takes (or the more you use it), the more this clock fills, and the attachment's condition changes. Once the clock is full, the item is either irredeemably lost or unusable until it is partially or fully repaired (Which can happen during downtime or through action rolls). If using the aforementioned repairing mechanic, you can have the repair clock be used as the item's condition clock, with each succesful repair clearing one (or multiple, your choice) segments.
```

Comment:
suggestion: I think you mean "introducing the `condition` mechanic"` and not consequence right?